app/backend/backend/settings.py

```
@@ -13,25 +13,33 @@
 from pathlib import Path
 from datetime import timedelta
 import os
-import environs
-
-env = environs.Env()
-env.read_env()
+from dotenv import load_dotenv
```

Pylint (3.3.7)

[error] Unable to import 'dotenv'

(E0401)